etc/adminhtml/system.xml

```diff
@@ -11,6 +11,11 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <config_path>payment/razorpay/active</config_path>
                 </field>
+                <field id="payment_action" translate="label" type="select" sortOrder="2" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
@ccrims0n I think it would be good to leave this option as is, because woocommerce has the same option as well.